src/ai/defaultai.cc

```diff
@@ -6783,7 +6783,8 @@ void DefaultAI::review_wares_targets(const Time& gametime) {
 	tribe_ = &player_->tribe();
 
 	// to avoid floats real multiplier is multiplier/10
-	const uint16_t multiplier = std::max<uint16_t>((productionsites.size() + num_ports * 5) / 5, 10);
+	const uint16_t multiplier =
+	   std::max<uint16_t>((productionsites.size() + static_cast<uint64_t>(num_ports) * 5) / 5, 10);
```

Comment:
casting to uint64_t while the final result as well as the other summand is only uint16_t might trigger the narrowing check not yet enabled and is errorprone